application/client-test/test/oasis-test.js

```diff
@@ -0,0 +1,52 @@
+const assert = require('assert');
+const config = require('../../config');
+const oasis = require('@oasislabs/client');
+const process = require('process');
```

Comment:
lol you went and moved the rest of the imports. Please use the following template

```
require(<items from node standard library>)

require(<items from npm modules>)

require(<local items>)
```
sort alphabetically within each group